tests/x11/multi_users_dm.pm

```diff
@@ -74,6 +74,8 @@ sub run {
         send_key 'ctrl-a';
         type_string "$user\n";
     }
+    # Make sure screen changed before calling handle_login function (for slow workers)
+    wait_still_screen;
```

Comment:
That's a bit sad, as we have `wait_screen_change` on L#67. The problem here is that we won't get still screen due to blinking cursor, so it's same as putting sleep for 7 seconds. With current approach, I don't see any better solution. The proper one is to remove needle that matches `displaymanager`, so we wait for `displaymanager-user-prompt`.

So what's about adding a parameter so that we handle it properly in `handle_login`? If there is any kind of urgency, let's merge this solution.

Reply:
@rwx788 not sure why wait_screen_change on L#67 is not enough. Maybe a quick  highlight when clicking the link?
What do you mean by *blinking cursor*? It is on graphic interface, not console.

It is now quite a long time we have this test failure, so we could merge it, and check later how to handle it differently..

Reply:
> Let's discuss point by point. So wait_screen_change fails to do it's job, because before and after click we have same screen, but we proceeded with code execution. Otherwise, we would not need to change anything. Blinking cursor is in the user name text field: https://openqa.opensuse.org/tests/761294#step/multi_users_dm/13
> There is a good solution to that, which will require much more efforts to do it properly and match our expectation. displaymanager needle is too generic. So in the end question is if you have time/will to make it in the way that we don't introduce yet another analog of sleep to sync test execution, or we start with this change. Sorry if I've confused you.

Ok, I misunderstood the 'blink' part. Now it is clear. Thanks.

I think we could rework the whole test to handle things in a better way, but let's start with this patch, for now, to get things fixed ASAP.

Reply:
Alright. Cheers ;)